app/services/licence-monitoring-station/setup/stop-or-reduce.service.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+/**
+ * Orchestrates presenting the data for `/licence-monitoring-station/setup/{sessionId}/stop-or-reduce` page
```

Comment:
In similar services this comment usually starts with `Orchestrates fetching and presenting`.